cmd/kops/create_cluster.go

```diff
@@ -159,6 +164,10 @@ func (o *CreateClusterOptions) InitDefaults() {
 	o.AdminAccess = []string{"0.0.0.0/0"}
 
 	o.Authorization = AuthorizationFlagRBAC
+
+	o.SpotinstProduct = "Linux/UNIX"
+	o.SpotinstOrientation = "balanced"
+	o.SpotinstDisableAutoScaler = false
```

Comment:
nit: I'd prefer to have those default values from a point responsible for default values. Defining them in create_cluster.go doesn't sound like the right place for me. Maybe put it into the API parts?

Reply:
[Done](https://github.com/spotinst/kubernetes-kops/blob/64176c7274f52af7408b28bf3be72cc399e3a7b1/upup/pkg/fi/cloudup/spotinsttasks/elastigroup.go#L1222-L1229).